helm_deploy/laa-estimate-eligibility/values/ccq-production.yaml

```diff
@@ -0,0 +1,90 @@
+replicaCount: 5
+
+resources: {}
+
+image:
+  repository: nginx
+  tag: stable
+  pullPolicy: IfNotPresent
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: false
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: "cd-serviceaccount"
+
+podSecurityContext: {}
+# fsGroup: 2000
+
+securityContext:
+ runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 80
+
+ingress:
+  hosts:
+    - host: main-ccq-production.cloud-platform.service.justice.gov.uk
+      secret: false
+    # - host: check-your-client-qualifies-for-legal-aid.cloud-platform.service.justice.gov.uk
+    #   secret: false
+    # - host: check-your-client-qualifies-for-legal-aid.service.gov.uk
+    #   secret: ccq-production-cert-secret
+  annotations:
+    external-dns.alpha.kubernetes.io/set-identifier: "check-client-qualifies-laa-estimate-eligibility-laa-check-client-qualifies-production-green"
+    external-dns.alpha.kubernetes.io/aws-weight: "0"
+
+postgresql:
+  enabled: false
+
+notifications:
+  errorMessageTemplateId: 7d7941c0-2f4d-43f2-acf8-12fdb8104e4d
+  recipient: patrick.gleeson@digital.justice.gov.uk,michael.blatherwick@digital.justice.gov.uk,william.clarke@digital.justice.gov.uk
+
+googleTagManager:
+  containerId: GTM-WJ98FR4
+
+cfe:
+  host: http://cfe-civil.cfe-civil-staging.svc.cluster.local
+  environment_name: production
+
+featureFlags:
+  sentry: ENABLED
+  overrides: NOT_ENABLED
+  indexProduction: NOT_ENABLED
```

Comment:
Can we add `# TODO:` comments to indexProduction and basicAuthentication to remind ourselves to switch them on/off respectively when we go live with this?